createmobilespec/createmobilespec.js

```diff
@@ -130,7 +135,7 @@ var top_dir =             process.cwd() + path.sep,
                    .boolean("windows").describe("windows", "Add Windows (universal) platform.")
                    .boolean("wp8").describe("wp8", "Add Windows Phone 8 platform.")
                    .boolean("firefoxos").describe("firefoxos", "Add FirefoxOS platform.")
-                   .boolean("plugman").describe("plugman", "Use {platform}/bin/create and plugman directly instead of the CLI.")
+                   .boolean("plugman").describe("plugman", "Use {platform}/bin/create and plugman directly instead of the CLI.") // TODO Why would one want to do that?
```

Comment:
I think this question should be sent to <dev@cordova.apache.org>, does not belong here.

My understanding is that `plugman` came before the more fancy Cordova CLI, and some people may not want to use the more advanced Cordova CLI. And I am now starting to wonder if developers could use plugman to add plugins when embedding Cordova in an existing mobile app?

Reply:
Yeah, my understanging on what `plugman` is and how it works (and how it relates to the other plugman call thing in tooling (lib I think it was)) is a moving target as well. 

I currently think it is to `cordova plugin add` as `cordova platform add` is to the platform `create` scripts. 

But we should really discuss this and see if someone else knows more - and then update the docs (and see where it pops up in all the tools and pieces (and maybe remove it there depending on our results)).

Reply:
```suggestion
                   .boolean("plugman").describe("plugman", "Use {platform}/bin/create and plugman directly instead of the CLI.")
```